pkg/storage/tree/profile.proto

```diff
@@ -38,9 +38,9 @@
 
 syntax = "proto3";
 
-package perftools.profiles;
+package perftools.profiles.tree;
 
-option go_package = "pkg/convert";
+option go_package = "pkg/tree";
```

Comment:
I'm not sure the whole `Profile` type should be moved to the `tree` package, moving only the specific Tree parser would have been enough I think, but maybe I'm missing something.

Reply:
This is again somewhat based on my opinion. The convert package only uses the single exported type `Profile` from the proto generated types, while the tree package uses a majority of its types. So the code is less noisy by keeping the types unprefixed. It also keeps the trio of related functions `ParseTreeNoDict`, `ParseIndividualLines`, `ParseGroups` in the same package.